internal/persistence/sql/namespaces.go

```diff
@@ -0,0 +1,87 @@
+package sql
```

Comment:
This file contains everything relevant for the mapping between the serial namespace ID and the UUID stored in the database.
Because it is expected to have few namespaces that are used all the time, I added a ristretto cache for that mapping.